source/_posts/react-input-validation.md

```diff
@@ -0,0 +1,121 @@
+---
+title: React Input Validation
+contributor: Gert J. Oberholzer
+avatar: kobus-oberholzer.png
+description: How to setup input validation that is both easy to use and secure
+date: 2023-02-13
+tags: [react, validation]
+id: react-input-validation
+relatedPost: react-Security-best-practices
+---
+
+{% img "banner.png" "React Input Validation" "lazy" %}
```

Comment:
This image can't be lazy. This is the first image which is always downloaded mobile/desktop. We can't defer it because we can lose LCP here stats for the post.